### PR TITLE
Feature/cover 37 forsiderdk vendor

### PR DESCRIPTION
**Pull request by turegjorup**
#### Link to ticket

https://jira.itkdev.dk/browse/COVER-37

#### Description

Depends on https://github.com/danskernesdigitalebibliotek/ddb-cover-service-importers/pull/143

- Added support for generic covers
- Added forsider.dk generic cover vendor
- Change `SingleIdentifier` vendors supports function to consider both type and identifier 

#### Screenshot of the result

If your change affects the user interface you should include a screenshot of the result with the pull request.

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

If you have any further comments or questions for the reviewer please add them here.

### CODE REVIEW

**Review by cableman — APPROVED** [review]

File: infrastructure/cover-service-cron-jobs/templates/cron-jobs.yaml [hunk 1]
```diff
@@ -88,6 +88,21 @@ spec:
...
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ .Release.Name }}-secret
+                      key: APP_FORSIDERDK_USER
+                - name: APP_FORSIDERDK_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ .Release.Name }}-secret
+                      key: APP_FORSIDERDK_PASSWORD
+                - name: APP_FORSIDERDK_SUBFOLDERS
```
Comment (cableman) [inline infrastructure/cover-service-cron-jobs/templates/cron-jobs.yaml:101]: This should no be here... if it's not a secrect. The whole configmap is loaded in another way

File: infrastructure/cover-service-cron-jobs/templates/secret.example.yaml [hunk 2]
```diff
@@ -19,6 +19,9 @@ stringData:
   APP_DB_USERNAME: ''
   APP_DB_PASSWORD: ''
   APP_THEMOVIEDATABASE_APIKEY: ''
+  APP_FORSIDERDK_USER: ''
+  APP_FORSIDERDK_PASSWORD: ''
+  APP_FORSIDERDK_SUBFOLDERS: ''
```
Comment (cableman) [inline infrastructure/cover-service-cron-jobs/templates/secret.example.yaml:24]: Not part off secret !

File: infrastructure/cover-service-importers/templates/import-deployment.yaml [hunk 3]
```diff
@@ -90,6 +90,21 @@ spec:
...
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Release.Name }}-secret
+                key: APP_FORSIDERDK_USER
+          - name: APP_FORSIDERDK_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Release.Name }}-secret
+                key: APP_FORSIDERDK_PASSWORD
+          - name: APP_FORSIDERDK_SUBFOLDERS
```
Comment (cableman) [inline infrastructure/cover-service-importers/templates/import-deployment.yaml:103]: Please see comment in cron deployment

File: infrastructure/cover-service-importers/templates/secret.example.yaml [hunk 4]
```diff
@@ -19,6 +19,9 @@ stringData:
   APP_DB_USERNAME: ''
   APP_DB_PASSWORD: ''
   APP_THEMOVIEDATABASE_APIKEY: ''
+  APP_FORSIDERDK_USER: ''
+  APP_FORSIDERDK_PASSWORD: ''
+  APP_FORSIDERDK_SUBFOLDERS: ''
```
Comment (cableman) [inline infrastructure/cover-service-importers/templates/secret.example.yaml:24]: Please see comment in cron deployment

File: infrastructure/cover-service-importers/templates/configmap.yaml [hunk 5]
```diff
@@ -31,6 +31,7 @@ data:
   APP_SUPERVISOR_QUEUES: {{ join " " .Values.app.supervisor.queues | quote }}
   APP_NO_HITS_TTL: {{ mul .Values.app.no_hits_ttl 1000 | quote }}
   APP_NO_HITS_CACHE_SINGLE_COVER_TTL: {{ mul .Values.app.no_hits_single_cover_ttl 1000 | quote }}
+  APP_FORSIDERDK_SUBFOLDERS: {{ .Values.app.vendor.forsiderdk_subfolders | quote }}
```
Comment (cableman) [inline infrastructure/cover-service-importers/templates/configmap.yaml:34]: Unsure about the format... `| qoute` her will "" the whole thing as an string... Not sure about helm and arrays

**Review by cableman — CHANGES_REQUESTED** [review]

File: infrastructure/cover-service-cron-jobs/templates/cron-jobs.yaml (hunk 1 above)
Comment (turegjorup) [reply infrastructure/cover-service-cron-jobs/templates/cron-jobs.yaml]: Removed

File: infrastructure/cover-service-importers/templates/configmap.yaml (hunk 5 above)
Comment (turegjorup) [reply infrastructure/cover-service-importers/templates/configmap.yaml:34]: I want the whole thing as a string. Symfony handles string -> array

**Pull request merged**